test/functional/normal/K_spec.lua

```diff
@@ -29,7 +27,7 @@ describe('K', function()
   it("invokes non-prefixed 'keywordprg' as shell command", function()
     helpers.source([[
       let @a='fnord'
-      set keywordprg=echo\ fnord\ >>]])
+      set keywordprg=echo\ fnord>>]])
```

Comment:
the extra space was intentional. does it cause a problem with cmd.exe?

Reply:
Output is `fnord ` with the extra space.